auth/user_mgt_test.go

```diff
@@ -496,6 +498,43 @@ func TestUpdateUser(t *testing.T) {
 		}
 	}
 }
+func TestRevokeRefreshTokens(t *testing.T) {
+	resp := `{
+		"kind": "identitytoolkit#SetAccountInfoResponse",
+		"localId": "expectedUserID"
+	}`
+	s := echoServer([]byte(resp), t)
+	defer s.Close()
+	before := time.Now().Unix()
+	err := s.Client.RevokeRefreshTokens(nil, "some_uid")
```

Comment:
Why are we passing a nil context? And why doesn't that trigger an error?

Reply:
This isn't making a network call.

Reply:
Doesn't it call the echo server?